ansible/roles/builder/prepare/tasks/setup_mock.yml

```diff
@@ -1,38 +1,9 @@
 ---
```

Comment:
Could we join this file with the `install_packages.yml` file? Both are small and could be together inside `main.yml`. This way we remove another "indirection layer".